.circleci/config.yml

```diff
@@ -0,0 +1,31 @@
+version: 2.1
+
+orbs:
+  codacy: codacy/base@2.2.1
+  
+jobs:
+  unit_test:
+    docker:
+      - image: circleci/golang:1.13
+    working_directory: ~/workdir
+    steps:
+      - checkout
```

Comment:
attach workspace instead

Reply:
It is better to attach indeed. Added it ✅